redalert/checks/run_bash_script.py

```diff
@@ -0,0 +1,35 @@
+# type: run-bash-script
+
+import subprocess
+import tempfile
+
+from .exc import CheckFailure
+
+class RunBashScript:
+    """A check for running a bash script and optionally checks output.
+
+    Takes the following arguments:
+    source (required): The source code of the script.
+    output: expected output string.
```

Comment:
nit: I would write this as "string which the output of the script will be compared to determine a successful run. If omitted only checks that the script exits with returncode 0"

The first sentence can stay the same as yours but I think we should mention the behavior when output is ommited